engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -188,12 +189,14 @@ class WorkflowManagerActor(config: Config)
   }
 
   private def pushToMetadataService(workflowId: WorkflowId): Unit = {
-    val curTime = currentTime
+    val curTime = OffsetDateTime.now
     val metadataEventMsgs = List(
       MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.Id), MetadataValue(workflowId.toString), curTime),
-      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.SubmissionTime), MetadataValue(curTime.asJodaString), curTime),
+      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.SubmissionTime),
+        MetadataValue(curTime.toString), curTime),
       // Currently, submission time is the same as start time
-      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.StartTime), MetadataValue(curTime.asJodaString), curTime)
+      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.StartTime), MetadataValue(curTime.toString),
```

Comment:
Is this functionally equivalent? I'm not sure whether consumers of the dates in metadata (esp. maybe the timing diagrams) relied on the `2008-09-18T00:00:00Z` format